collectors/python.d.plugin/logparser/logparser.conf

```diff
@@ -0,0 +1,39 @@
+#This module is able to monitor an application specific log file and then create a chart based on occurrences of a log line (like amount of occurrences in a day, or a line graph of when the occurrences happen).It has no limitation to have multi dimension and multi charts.
+
+# Config patterns
+
+#```yaml
+#chart_name:
+#    log_path: path/log/file
+#    dimensions:
+#      dimension: regex_pattern
+#```
+
+#Above config show how to define your charts and how to fetching metrics from custom log files.
+#For each dimension in each chart must one regex be written in order to fetch those matches from that log file.It allows us to define whatever charts we want to show in dashboard.
+
+#A final config for more than one chart and more than one dimension could be something like this
+
+#```yaml
+
+#chart1_name:
+#    log_path: /path/logs/log.file
+#    dimensions:
+#      dimension_name1: 'string=GET'
+#      dimension_name2: 'string=POST'
+#      dimension_name3: 'string=PATCH'
+#chart2_name:
+#    log_path: /path/logs/log2.file
+#    dimensions:
+#      dimension_name1: 'regexp=[A-Z]+'
+#      dimension_name2: 'regexp=[A-Z]+'
+#      dimension_name3: 'regexp=[A-Z]+'
+#
+#```
+
+chart1_name:
+    log_path: /path/logs/log.file
+    dimensions:
+      bar1: 'string=GET'
+      bar2: 'string=^GOT'
+      bar3: 'regexp=G[QWERTY]T'
```

Comment:
i see no reason to have this job not comment out

Reply:
It's just a sample, you think we do not any uncomment sample there, do you???

Reply:
It makes no sense to me, module will try to open `/path/logs/log.file`

We don't need predefined jobs.

Add as much samples as you want, but please comment out them

Reply:
We don't need predefined jobs. Samples are ok.

Add as much samples as you want, but please comment out them